samples/service.js

```diff
@@ -55,11 +93,182 @@ function checksum (str, algorithm = 'sha256', encoding = 'hex') {
     .digest(encoding)
 }
 
-function arrayBufferToBuffer (ab) {
-  var buf = new Buffer(ab.byteLength)
-  var view = new Uint8Array(ab)
-  for (var i = 0; i < buf.length; ++i) {
-    buf[i] = view[i]
+function readId3Tags (file) {
+  return new Promise((resolve, reject) => {
+    JsMediaTags.read(file, {
+      onSuccess: resolve,
+      onError: reject
+    })
+  })
+}
+
+const SAMPLE_RATE = 44100
```

Comment:
@ahdinosaur don't review this file from here on. this is all utils i ported from old linx, works as expected